client/src/components/menubar/index.js

```diff
@@ -362,30 +316,43 @@ class MenuBar extends React.Component {
         }}
       >
         {this.renderDiffExp()}
-        <Tooltip
-          content="Show only metadata and cells which are currently selected"
-          position="bottom"
-          hoverOpenDelay={globals.tooltipHoverOpenDelay}
-        >
-          <AnchorButton
-            data-testid="subset-button"
-            disabled={
-              crossfilter &&
-              (crossfilter.countSelected() === 0 ||
-                crossfilter.countSelected() === crossfilter.size())
-            }
-            style={{
-              marginRight: 10
-            }}
-            onClick={() => {
-              dispatch(actions.regraph());
-              dispatch({ type: "increment graph render counter" });
-            }}
+        <ButtonGroup style={{marginRight: "10px"}}>
+          <Tooltip
+            content="Subset to currently selected cells and associated metadata"
+            position="bottom"
+            hoverOpenDelay={globals.tooltipHoverOpenDelay}
           >
-            <Icon icon="double-chevron-down" />
-          </AnchorButton>
-        </Tooltip>
-        <ButtonGroup style={{ marginRight: "10px" }}>
+            <AnchorButton
```

Comment:
suggestion - factor this out into a sub-component ans file like Clip or InformationMenu